app/models/driver.rb

```diff
@@ -0,0 +1,33 @@
+class Driver < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: { message: "Name cannot be blank!"}
+  validates :vin, presence: { message: "VIN cannot be blank!"}
+
+  paginates_per 10
+
+  def total_money
+    total = super.to_f
+    total = total - (total * 0.15)
+    return '%.2f' % total
+  end
+
+  def average_rating
+    if trips.any? == false
+      return "Nil"
+    end
+    length = trips.length
+    total = 0
+
+    trips.each do |trip|
```

Comment:
You could also use an enumerable method to pluck out the ratings values that were populated and create an average that way